sys/universal_address/universal_address.c

```diff
@@ -28,7 +34,17 @@
 /**
  * @brief Maximum number of entries handled
  */
-#define UNIVERSAL_ADDRESS_MAX_ENTRIES (40)
+/* determine the maximum numer of entries */
+#ifndef UNIVERSAL_ADDRESS_MAX_ENTRIES
```

Comment:
@OlegHahm What's your opinion on moving this `ifdefing` into a separate file and `#include` it here? Maybe this kind of `ifdef`ing can be reused somewhen somewhere else..  It also would keep this `.c` file clean and you wouldn't need to include the headers at the very top